scripts/scr_ui_advisors/scr_ui_advisors.gml

```diff
@@ -1,19 +1,61 @@
 function scr_ui_advisors() {
 
-	var xx,yy, blurp, eta;
+	var xx,yy, blurp, eta, va;
+	var romanNumerals;
+	romanNumerals=scr_roman_numerals();
+	var recruitment_rates = [
+    	"sluggish",
+    	"slow",
+    	"moderate",
+    	"fast",
+    	"frenetic",
+    	"hereticly fast"
```

Comment:
if these are not localisation strings but string ids, then I highly recommend using _ instead of spaces, will make our life easier once we switch to json